addon/components/nypr-accounts/membership-card/donation-button.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+import layout from '../../../templates/components/nypr-accounts/membership-card/donation-button';
+
+export default Ember.Component.extend({
+  layout,
+  pledgePrefix: Ember.computed(function() {
+    let { environment } = Ember.getOwner(this).resolveRegistration('config:environment');
+    return environment === 'development' ? 'pledge-demo' : 'pledge3';
```

Comment:
this is also something that should be addressed in the future... the addon shouldn't need to make value judgements based on its environment